### PR TITLE
Fix CSS for choice elements in outline docs

### LINKED ISSUE #1365: Fix CSS for choice elements in outline docs
### User Story

As an OSCAL tools developer, in order to better understand the use of `choice` in outline documentation, I want better layout so I can better understand hierarchical options of `choice`s and their relationships to sibling elements that are siblings (even though they look like children).

### Goals

We want to adjust the `choice` so, per the example below, `condition` and `test-reference` are on the same level of the hierarchy as `pre-condition` not indented as children of it.

oscal_reference_docs_choice_layout

### Dependencies

_No response_

### Acceptance Criteria

- [x] All [OSCAL website](https://pages.nist.gov/OSCAL) and readme documentation affected by the changes in this issue have been updated. Changes to the OSCAL website can be made in the docs/content directory of your branch.
- [x] A Pull Request (PR) is submitted that fully addresses the goals of this User Story. This issue is referenced in the PR.
- [x] The CI-CD build process runs without any reported errors on the PR. This can be confirmed by reviewing that all checks have passed in the PR.

### PR DESCRIPTION
**Pull request by nikitawootten-nist**
# Committer Notes

Fixes #1365

Choices are now displayed in outlines as follows:

JSON view:
Screenshot 2023-03-02 at 2 05 14 PM

XML view:
Screenshot 2023-03-02 at 2 06 10 PM

Nested choices alternate between white (the background color) and light gray up to a depth of 3 (I assume if things get deeper then 3 levels of choices, we have other problems)

This change has been tested on both Firefox and Chrome.

## UPDATE 3/3/23

@aj-stein-nist suggested changing the color to fit our palette, as well as re-adding a bit of indentation:

image

### All Submissions:

- [x] Have you selected the correct base branch per [Contributing](https://github.com/usnistgov/OSCAL/blob/main/CONTRIBUTING.md) guidance?
- [x] Have you set "[Allow edits and access to secrets by maintainers
](https://docs.github.com/en/pull-requests/collaborating-with-pull-requests/working-with-forks/allowing-changes-to-a-pull-request-branch-created-from-a-fork)"?
- [x] Have you checked to ensure there aren't other open [Pull Requests](https://github.com/usnistgov/OSCAL/pulls) for the same update/change?
- [x] Have you squashed any non-relevant commits and commit messages? \[[instructions](https://git-scm.com/book/en/v2/Git-Tools-Rewriting-History)\]
- [x] Do all automated CI/CD checks pass?

By submitting a pull request, you are agreeing to provide this contribution under the [CC0 1.0 Universal public domain](https://creativecommons.org/publicdomain/zero/1.0/) dedication.

### Changes to Core Features:

- [x] Have you added an explanation of what your changes do and why you'd like us to include them?
- [x] Have you written new tests for your core changes, as applicable?
- [x] Have you included examples of how to use your new feature(s)?
- [x] Have you updated all [OSCAL website](https://pages.nist.gov/OSCAL) and readme documentation affected by the changes you made? Changes to the OSCAL website can be made in the docs/content directory of your branch.

### CODE REVIEW

**Comment by nikitawootten-nist** [comment]
Relevant lines live on lines 154-173

**Comment by aj-stein-nist** [comment]
I will take this for a spin later tonight or tomorrow, personally I am preferential to Option 3. Would like to know what the rest of the team thinks.

**Review by wendellpiez — APPROVED** [review]
Approving so as not to block improvement - while a more thorough redesign/shakeout of CSS may *also* be in order (some day) ...

**Comment by nikitawootten-nist** [comment]
> I will take this for a spin later tonight or tomorrow, personally I am preferential to Option 3. Would like to know what the rest of the team thinks.

For those reviewing this, Option 3 refers to the mock ups I posted [here](https://github.com/usnistgov/OSCAL/issues/1365#issuecomment-1452230300).

Personally Option 3 looks a little confusing to me, but if the team would rather a more muted line instead of coloring the whole background, I can probably improve the design as well.

**Comment by aj-stein-nist** [comment]
> > I will take this for a spin later tonight or tomorrow, personally I am preferential to Option 3. Would like to know what the rest of the team thinks.
> 
> For those reviewing this, Option 3 refers to the mock ups I posted [here](https://github.com/usnistgov/OSCAL/issues/1365#issuecomment-1452230300).
> 
> Personally Option 3 looks a little confusing to me, but if the team would rather a more muted line instead of coloring the whole background, I can probably improve the design as well.

I was going to say: you like the boxes!? But if you want more muted lines I could understand. That said: I think the most key thing is a very clear visual identifier that shows the indentation or "hierarchy" of the objects in the model, and the box options make that less obvious to me personally. But it seems we are in the same head space.

So we like the line concept, but could make it better, is that the takeaway?

**Comment by wendellpiez** [comment]
I actually like the lines, but I also wonder if the indents shouldn't be at the level down, i.e. the choices. A left-side rule might be made to span across them. So the p would in effect look like a list header. (Unfortunately we don't have yet another wrapper around the group of choice divs.)

However I am also not inclined to quibble here, as they are all improvements to legibility.

**Review by aj-stein-nist — COMMENTED** [review]
I am of mixed my minds of whether to point this change to `develop` or `main` given it's a site change, but I guess we should stick with `develop` for now.

File: docs/assets/scss/schema-docs.scss [hunk 1]
```diff
@@ -81,68 +89,99 @@
...
+    div.OM-choices {
+      background-color: white;
+
+      div.OM-choices {
+        background-color: lightgray;
+
+        // Add nested colors as needed
+      }
+    }
   }
```
Comment (aj-stein-nist) [inline docs/assets/scss/schema-docs.scss:174]: How about this? A blue closer to our current palette and more aggressive indentation?

image

image


```suggestion
  div.OM-choices {
    @include u-margin-left(-2);
    @include u-padding-left(2);
    background-color: #d9e8f6;
    width: max-content;

    div.OM-choice {
      @include u-padding-left(4);
    }

    // Nested choices alternate color for readability
    div.OM-choices {
      background-color: white;

      div.OM-choices {
        background-color: lightgray;
        // Add nested colors as needed
      }
    }
  }
```

**Review by aj-stein-nist — APPROVED** [review]
:ship: it.

**Pull request merged**